90anti-evil-maid/anti-evil-maid-unseal

```diff
@@ -130,10 +131,10 @@ fi
 # check freshness token
 
 log "Unsealing freshness token..."
-if tpm_unsealdata $Z -i "$SEALED_SECRET_FRE" -o "$UNSEALED_SECRET" \
+if tpm_unsealdata "$Z" -i "$SEALED_SECRET_FRE" -o "$UNSEALED_SECRET" \
```

Comment:
This one (and few similar below) was intentionally not quoted - if no `-z` is intended, it should give no argument, not empty argument.

Reply:
I thought it might be, undone changes with `$Z` and suppressed warnings.